data/types/base/base_test.go

```diff
@@ -48,26 +48,26 @@ import (
 // TODO_DATA: Need to find another way to test base not using sample/sub
```

Comment:
maybe we test it as part of another type and just concentrate on the base fields? better than no tests